contrib/Zipkin/Exporter.php

```diff
@@ -51,8 +64,9 @@ public function __construct($name, string $endpointUrl, SpanConverter $spanConve
         }
 
         $this->endpointUrl = $endpointUrl;
-
+        $this->client = $client ?? $this->createDefaultClient();
         $this->spanConverter = $spanConverter ?? new SpanConverter($name);
+        $this->client = $client;
```

Comment:
It looks like this line will over-ride the null-coalesce on line 67?
When `$client` is `null`, line 67 will set `$this->client` to `$this->createDefaultClient()` and then line 69 will set `$this->client` to `null`

Reply:
you are right, I will fix it right away.

Reply:
done